Branch-SDK/src/main/java/io/branch/referral/Defines.java

```diff
@@ -160,12 +160,6 @@ public enum Jsonkey {
         SearchQuery("search_query"),
         AdType("ad_type"),
 
-        // to accommodate Tune -> Branch migration
-        CPUType("cpu_type"),
```

Comment:
@gdeluna-branch we need Justin's input here. While those were introduced originally for Tune, I think others may rely on it right now.